examples/twittermap/web/public/javascripts/timeseries/controllers.js

```diff
@@ -37,7 +37,7 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
     countDiv.title = "Display the count information of Tweets";
     countDiv.innerHTML = [
       "<div ng-if='queried'><p id='count'>{{ currentTweetCount | number:0 }}<span id='count-text'>&nbsp;&nbsp;of&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p></div>",
-      "<p id='count'>{{ totalCount | number:0 }}<span id='count-text'>&nbsp;&nbsp;{{sumText}}</span></p>",
+      "<p id='count'>{{ totalCount | number:0 }}<span id='count-text'>{{sumText}}</span></p>",

```

Comment:
this way "mentions" gets too close to the total count. Right ?

Reply:
That is not a proper way of spacing elements as you can't control it by css. The idea here, anyways, is to have the span lying above the count, so the horizontal space between them won't really matter anymore.

Reply:
ok then